### DISCUSSION TITLE
Apexcharts RadialBar height is wrong, how can i set it right?

### DISCUSSION THREAD

**Discussion by aboutjquery** [apexcharts/apexcharts.js#3157] [Q&A]
Today I tried Apexchart RadialBar and other charts, some height settings are inaccurate, and it's worse in RadialBar.

This is an example of the official website of the arch https://codepen.io/junedchhipa/pen/eQmmbO

You can see that the set height 350 does not work, it is completely inaccurate, is there a way to set the specified height? e.g. 250 height
```

var options = {
  chart: {
      height: 350,
      type: 'radialBar',
  },
  series: [70],
  labels: ['Progress'],
}

var chart = new ApexCharts(document.querySelector("#chart"), options);

chart.render();
```

**Comment by MarioPerini** [comment]
Yes set the hight on the parent and set the height of the chart to '100%'
https://codepen.io/MarioPerini/pen/YzexBGR
